bundles/org.openhab.binding.mybmw/src/test/java/org/openhab/binding/mybmw/internal/dto/StatusWrapper.java

```diff
@@ -248,8 +248,8 @@ private void checkResult(ChannelUID channelUID, State state) {
                 assertEquals(expected.toString(), dtt.toString(), "Last Update");
                 break;
             case GPS:
-                if (state instanceof PointType) {
-                    pt = (PointType) state;
+                if (state instanceof PointType point) {
+                    pt = point;
```

Comment:
```suggestion
                if (state instanceof PointType pt) {
```

Reply:
@jlaur this would hide the previously defined local variable pt.
In this file, all temp variables are declared in front of the case block (L106-112).
So I am not sure how to proceed...

Reply:
Sorry, you are right, I missed this.